src/SFA.DAS.FindApprenticeshipTraining/InnerApi/Responses/GetStandardsListItem.cs

```diff
@@ -31,39 +31,67 @@ public class GetStandardsListItem
         public string IntegratedDegree { get; set; }
         public string SectorSubjectAreaTier2Description { get; set; }
         public decimal SectorSubjectAreaTier2 { get; set; }
-        private long GetMaxFunding()
+
+        private long GetFundingDetails(string prop)
         {
             var funding = ApprenticeshipFunding
-                .FirstOrDefault(c => 
-                    c.EffectiveFrom <= DateTime.UtcNow && (c.EffectiveTo == null 
+                .FirstOrDefault(c =>
+                    c.EffectiveFrom <= DateTime.UtcNow && (c.EffectiveTo == null
                                                            || c.EffectiveTo >= DateTime.UtcNow));
+            if (prop == "MaxFunding")
```

Comment:
i think you can probably keep all of this in one method, there is duplication in those two methods where all it is trying to do is to find a ApprenticeshipFunding item that meets the criteria